### PR TITLE
RU pack

### PR DESCRIPTION
**Pull request by powerdot**
## Type of Change
RU i18l changes: sounds better, UI looks greater.

## What issue does this relate to?
Continued localization of the application.

### What should this PR do?
Better localization!

### CODE REVIEW

File: src/nginxconfig/i18n/ru/templates/domain_sections/server.js [hunk 1]
```diff
@@ -29,9 +29,9 @@ export default {
     documentRoot: 'Корневая директория',
     oneOrMoreOtherDomainsAreAlsoNamed: 'Один или несколько других доменов также названы',
     thisWillCauseIssuesWithConfigGeneration: 'Это вызовет проблемы с генерацией конфигурации.',
-    wwwSubdomain: 'www поддомен',
+    wwwSubdomain: 'WWW поддомен',
```
Comment (powerdot) [inline src/nginxconfig/i18n/ru/templates/domain_sections/server.js:32]: (Capitalized `www` is a strange Russian tradition.)

File: src/nginxconfig/i18n/ru/templates/domain_sections/server.js [hunk 2]
```diff
@@ -29,9 +29,9 @@ export default {
...
     cdnSubdomain: 'CDN поддомен',
     redirectSubdomains: 'Перенаправлять поддомены',
     server: 'Сервер',
-    listen: 'слушать',
+    listen: 'Слушать от адреса',
```
Comment (powerdot) [inline src/nginxconfig/i18n/ru/templates/domain_sections/server.js:36]: I checked where this translation is used - it is best used here for better understanding.

File: src/nginxconfig/i18n/ru/templates/global_sections/logging.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ THE SOFTWARE.
 import common from '../../common';
 
 export default {
-    enableFileNotFoundErrorLogging: `${common.enable} файл не найден, ошибка входа`,
+    enableFileNotFoundErrorLogging: `${common.enable} логирование ошибок для файлов, которые не были найдены при запросе`,
```
Comment (powerdot) [inline src/nginxconfig/i18n/ru/templates/global_sections/logging.js:30]: I know this string looks very huge, but I verified it with UI - all is ok!

**Review by MattIPv4 — APPROVED** [review]
Lgtm, ty!

**Pull request merged**